addons/stock/models/product.py

```diff
@@ -593,6 +601,20 @@ def action_open_quants(self):
         action['context'] = {'search_default_internal_loc': 1}
         return action
 
+    def action_view_related_putaway_rules(self, **params):
```

Comment:
we don't usually use **params in Odoo + i don't think it's the proper way to do things in a method called by a button in a form

pleae remove this arg and create a private method to return the action as per my previous comment
thank you :)